### ISSUE TITLE
Move search API off Heroku

### ISSUE THREAD

**Issue by ayaleloehr** [department-of-veterans-affairs/veterans-employment-center#118]
This has to happen before 2/7. On that date Heroku is shutting off free access because we are over our quota.

**Comment by ayaleloehr** [comment]
Changed a few add-ons on Heroku to save us money overall while getting us back under the search index size limits. Still pursuing moving to AWS, but 2/7 date is no longer the driving factor.  Current blocker is waiting on USAJOBS to whitelist new IP addresses for us.

**Comment by ayaleloehr** [comment]
USAJOBS says they are still waiting to hear back from their security on whitelisting the new IPs.

**Comment by koconnor719** [comment]
high priority

**Comment by koconnor719** [comment]
not a high priority as of 3/12 per alex (no longer cost issues; high priority mitigated) ; team does NOT want to move right now; close issue and will reopen when relevant again.

**Comment by MarinaMartin** [comment]
I don't think we should close this issue or we will forget about it. We should just leave it in the backlog.

**Comment by ayaleloehr** [comment]
@MarinaMartin agreed. That's what we did (even though we wrote otherwise in the above comment).

**Comment by joduinn** [comment]
@ayaleloehr and myself noted that an intermittent alert in prod-vec-elb in AWS is being caused by an intermittent delay on heroku. Alerts tweaked, but this revived the discussion about moving from heroku to aws. 

Not *urgent*, but noting for the record and will decide timing after we decide on a region in AWS.

**Comment by joduinn** [comment]
@ayaleloehr confirmed that some recent intermittent alerts in prod-vec-elb are being caused by this same intermittent delay on heroku. 

For now, tweaking alerts, but we need to remember to ratchet down these alerts later, after we move off heroku, hence updating this ticket.
